### ISSUE TITLE
Jupyterlab 3.0.11 does not load in Safari 14.0.3

### ISSUE THREAD

**Issue by jbulow** [jupyterlab/jupyterlab#9973]
When starting jupyter lab safari show a blank page. Pasting the same URL in chrome works fine.

The error in safari dev console is:

```
[Error] SyntaxError: Invalid regular expression: invalid group specifier name
	(anonymous function) (jlab_core.72d583df05e5e4e3d8d7.js:2)
[Error] Unhandled Promise Rejection: ChunkLoadError: Loading chunk 3472 failed.
(missing: http://localhost:8888/static/lab/jlab_core.72d583df05e5e4e3d8d7.js?v=72d583df05e5e4e3d8d7)
	(anonymous function) (main.a4e1852903a39d82f389.js:1:1034)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
	promiseReactionJob
```

**Comment by jtpio** [comment]
Thanks @jbulow for reporting.

This should normally be fixed in `3.0.12`: https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html#v3-0-12

Are you able to update to `3.0.12` to see if it fixes the issue?

**Comment by jasongrout** [comment]
I confirmed the error in Safari in 3.0.11, and confirmed that 3.0.12 works in Safari.

@jbulow, please comment if 3.0.12 if you are seeing this error with jlab 3.0.12, and we can reopen the issue.
